controllers/progressivesync_controller_test.go

```diff
@@ -29,60 +29,50 @@ const (
 	interval = time.Millisecond * 10
 )
 
-var _ = Describe("ProgressiveSync Controller", func() {
+var (
+	appSetAPIRef = utils.AppSetAPIGroup
+	ctx          = context.Background()
+)
 
-	var (
-		ctx                     context.Context
-		namespace, appSetAPIRef string
-		ns                      *corev1.Namespace
-		ownerPR, singleStagePR  *syncv1alpha1.ProgressiveSync
-	)
+func createRandomNamespace() (string, *corev1.Namespace) {
+	namespace := "progressiverollout-test-" + randStringNumber(5)
+
+	ns := &corev1.Namespace{
```

Comment:
```suggestion
	ns := corev1.Namespace{
```
as per the convention introduced in https://github.com/Skyscanner/applicationset-progressive-sync/pull/85

Reply:
good catch